### ISSUE TITLE
Como proceder com a instalação?

### ISSUE THREAD

**Issue by rafaelsabino** [praxd/bolaocopa#1]
Pode me dizer por favor como proceder com a instalação?

**Comment by praxd** [comment]
Rafael, até o momento foram adicionados somente os arquivos do Laravel, o projeto ainda será desenvolvido

**Comment by rafaelsabino** [comment]
Certo, muito obrigado então. Procurando a tempos um sistema de apostas pra amigos da faculdade e o seu pareceu bem interessante. Vlew mano.

**Comment by praxd** [comment]
Quando eu terminar a implementação te mando uma mensagem 😃

**Comment by rafaelsabino** [comment]
Obrigado mano! Vlew mesmo
